tests/storage/redis/redis_store.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the Redis storage."""
+
+from __future__ import unicode_literals
+
+import fakeredis
+import redis
+
+from plaso.containers import events
+from plaso.containers import sessions
+from plaso.containers import tasks
+from plaso.storage.redis import redis_store
+from plaso.storage.redis import writer
+
+from tests.containers import test_lib as containers_test_lib
+from tests.storage import test_lib
+
+
+class RedisStoreTest(test_lib.StorageTestCase):
+  """Tests for the Redis storage object."""
+
+  # pylint: disable=protected-access
+
+  def _GetRedisClient(self):
+    """Creates a Redis client for testing.
+
+    This method will attempt to use the
+
+    Returns:
+      Redis: a Redis client.
+    """
+    try:
+      redis_client = redis.from_url('redis://127.0.0.1/0', socket_timeout=60)
```

Comment:
Question: do you want to fill a local redis with plaso test data? or should we always use fake here and leave the real implementation tests for the e2e tests ?

Reply:
My initial thought was to do things this way for the unite tests on Travis, since it's pretty easy to have a real Redis there, but for developers on their local machines they shouldn't need to set up a server just to run the tests. On balance, I think I still prefer it like this, but if you have a strong view about this I'm open to changing the code.

Reply:
I was thinking it could lead to some surprises of having plaso unittest data  in your local redis server. Let's keep this as-is for now, and considered controlling this behavior via an environment variable that can be set on Travis and other CI environments.